ui.tests/test-module/specs/recaptcha/recaptcha.runtime.spec.js

```diff
@@ -112,35 +113,129 @@ describe("Form Runtime with Recaptcha Input", () => {
+            cy.on('window:alert', (message) => {
+                expect(message).to.equal('Encountered an internal error while submitting the form.');
+            });
+            cy.intercept('POST', '/adobe/forms/af/submit/*').as('submitForm');
+            cy.get(`div.g-recaptcha iframe`).should('be.visible').then(() => {
+                const [id, fieldView] = Object.entries(formContainer._fields).find(([id, fieldView]) => id.includes("captcha"));
+                const model = formContainer._model.getElement(id);
+                cy.get(`#${id}`).then(x => {
+                    model.value = "dummyResponseToken";
+                    cy.get(`.cmp-adaptiveform-button__widget`).click().then(x => {
+                        cy.wait('@submitForm').then((interception) => {
+                            expect(interception.response.statusCode).to.equal(400);
+                            expect(interception.response.body).to.have.property('title', 'The CAPTCHA validation failed. Please try again.');
+                        });
+                    });
+                })
+            });
+        }
+    })
+
+
     it("submission should pass for enterprise score based captcha",() => {
         updateEnterpriseConfig(0.5);
         cy.previewForm(enterprisePagePath).then((p) => {
             formContainer = p;
         });
         expect(formContainer, "formcontainer is initialized").to.not.be.null;
         cy.get(`div.grecaptcha-badge`).should('exist').then(() => {
-            cy.get(`.cmp-adaptiveform-button__widget`).click().then(x => {
-                // Need to submit multiple times until the form is submitted successfully
-                // Due to below error while validating recaptcha enterprise
-                // https://cloud.google.com/recaptcha-enterprise/docs/faq#returned_browser_error_when_creating_an_assessment_what_should_i_do_about_this
-                cy.waitUntil(() =>
-                    cy.location('href').then((href) => {
-                        if (href.includes("guideContainer.guideThankYouPage.html")) {
-                            return true;
-                        }
-                        else {
-                            cy.get(`.cmp-adaptiveform-button__widget`).click().then(() => {
-                                return false;
-                            });
-                        }
-                    }),
-                    {
-                        errorMsg: 'The form with recaptcha enterprise was not submitted successfully',
-                        timeout: 50000,
-                        interval: 5000
+            cy.intercept('POST', '/adobe/forms/af/submit/*').as('submitForm');
+            const submitForm = () => {
+                cy.get(`.cmp-adaptiveform-button__widget`).click();
+
+                // Wait for the submitForm request
+                return cy.wait('@submitForm',{ timeout: 50000 }).then((interception) => {
```

Comment:
Can we remove the timeout from here ?

Reply:
For the local execution, cypress was not getting the submitForm request initiated within the default timeout. 
So, I added the time timeout for the same, however in the circle ci I can see that this executes much faster. 
If I remove the timeout the test might fail.